src/Components/Auth/SignUp/signup.js

```diff
@@ -0,0 +1,90 @@
+import "../SignUp/signup.css";
+import { useForm } from "../../../custom-hooks/useForm";
+import { useAuth } from "../../../Context/authContext";
+const SignUp = () => {
+  const { formData, errors, formHandler } = useForm();
+  const { signUpHandler } = useAuth();
+  return (
+    <>
+      <div class="container-signup">
+        <div class="left">
+          <h1>Sign Up</h1>
+          <p>create a new account</p>
+          <form className="signup-form" onSubmit={(e) => e.preventDefault()}>
+            <label for="firstName" className="lable">
+              First Name
+            </label>
```

Comment:
`for` should not be used in react as `for` is a reserved keyword in js you should use `htmlFor` instead of it